### ISSUE TITLE
Headers are not accessible in Android Accessibility (Flutter Gallery)

### ISSUE THREAD

**Issue by goderbauer** [flutter/flutter#10970]
headers ("Demos", "Material Components", etc) are not selectable in accessibility mode on Android and therefore are not read out loud. It works fine on iOS.

**Comment by goderbauer** [comment]
Interesting: When you swipe through all accessibility components at the very end (or beginning?) the entire screen gets selected and "Flutter Gallery, Demos, Material Components" is read out loud. That seems wrong.


Also: the very top heading "Flutter Gallery" is also not selectable on its own.

**Comment by goderbauer** [comment]
This has been fixed.
